bin/configInitializer.js

```diff
@@ -1,57 +0,0 @@
-const fs = require('fs');
```

Comment:
This file was renamed `mockFileReader.js`, and then some minor refactoring were done.